.travis.yml

```diff
@@ -64,6 +64,11 @@ matrix:
       env:
         - "JRUBY_OPTS='--dev -J-Xmx1024M'"
         - "GEM='ap'"
+    - rvm: jruby-9.1.5.0
+      jdk: oraclejdk8
+      env:
+        - "JRUBY_OPTS='--dev -J-Xmx1024M'"
```

Comment:
Can we move this somewhere more global, before it proliferates too much? It seems likely to change (so copying is bad), but it also just gets in the way while reading the build matrix.